requirements_for_test.txt

```diff
@@ -1,7 +1,7 @@
 -r requirements.txt
 
-black==23.10.1  # Also update `.pre-commit-config.yaml` if this changes
-ruff==0.0.275  # Also update `.pre-commit-config.yaml` if this changes
+black==24.4.0  # Also update `.pre-commit-config.yaml` if this changes
+ruff==0.3.7  # Also update `.pre-commit-config.yaml` if this changes
```

Comment:
Need to also update [`.pre-commit-config.yaml`](https://github.com/alphagov/document-download-frontend/blob/main/.pre-commit-config.yaml)